### ISSUE TITLE
[Enhancement] Add support for Stripe.

### ISSUE THREAD

**Issue by NurdTurd** [cdmedia/simplestore#21]
I think Stripe would be a good payment method to add next. Tell me what you guys think.

Website: https://stripe.com

**Comment by NurdTurd** [comment]
Well, I just realized the close issue button closes the issue without warning. :smile:

**Comment by 88kbbq** [comment]
Yes, Stripe would be fantastic. Better experience than Paypal, plus Paypal is blocked in some Asian countries.
